sidebar.js

```diff
@@ -1,26 +1,37 @@
+let sidebarTemplateContent
+
+async function loadSidebarTemplate () {
```

Comment:
We can just call this code without wrapping it in a function since modules have top level await these days.